examples/testmonitor/fileattachment/FileAttachment.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using NationalInstruments.SystemLink.Clients.TestMonitor;
+using NationalInstruments.SystemLink.Clients.File;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.TestMonitor
+{
+    /// <summary>
+    /// Example for the SystemLink Test Monitor API that creates a test result,
+    /// uploads a file to the SystemLink server, and attaches the file to the
+    /// test result.
+    /// </summary>
+    class FileAttachment
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * See the configuration example for how a typical application
+             * might obtain a configuration.
+             */
+            var configuration = ExampleConfiguration.Obtain(args);
```

Comment:
Probably true for the previous Test Monitor example as well, but I just realized we should be doing this:
```suggestion
            var configuration = ExampleConfiguration.Obtain(args, allowCloud: false);
```